DX-978: Add Payment Menu section (#444)
SwedbankPay/developer.swedbankpay.com: payment-menu/index.md

---

> title: Swedbank Pay Payment Menu– Introduction

Reviewer: ```suggestion
title: Swedbank Pay Payment Menu – Introduction
```

sidebar:
  navigation:
  - title: Payment Menu
    items:
    - url: /payment-menu/
      title: Introduction
    - url: /payment-menu/payment-order
      title: Payment Order
    - url: /payment-menu/capture
      title: Capture
    - url: /payment-menu/after-payment
      title: After Payment
    - url: /payment-menu/other-features
      title: Other Features
---

> {% include jumbotron.html body="**Swedbank Pay Payment Menu** integrates
> seamlessly into the merchant website through highly customizable and flexible
> components."%}

Reviewer: ```suggestion
{% include jumbotron.html body="The **Swedbank Pay Payment Menu** integrates
seamlessly into the merchant website, allowing the payer to choose between the
increasing number of payment instrument Swedbank Pay has on offer and
remembering which instrument they prefer and have paid with previously.
```

Swedbank Pay Payment Menu allows your customers to pay with their favorite payment

> methods in just a few simple steps.

Reviewer: ```suggestion
instruments in just a few simple steps.
```

## Prerequisites

To start integrating Swedbank Pay Payment Menu, you need the following:

*   [HTTPS][https] enabled web server.
*   Agreement that includes Swedbank Pay Payment Menu.
*   Obtained credentials (merchant Access Token) from Swedbank Pay through
  Swedbank Pay Admin.

## Introduction

To get started with Swedbank Pay Payment Menu, you should learn about its different
components and how they work together. **Payment Menu** authorizes the
payment with our Payment Menu API. The next step is to **Capture** the payment.
You can either capture the total amount, or do a part-capture (as described
under [After Payment][after-payment-capture]). Connect these steps and you have
Swedbank Pay Payment Menu.

> Under, you will see a sequence diagram showing the sequence of a Swedbank Pay

Reviewer: ```suggestion
Below, you will see a sequence diagram showing the sequence of a Swedbank Pay
```

            activate Payer
            deactivate Payer
            Merchant ->>+ SwedbankPay: GET rel: paid-paymentorder
            deactivate Merchant
            SwedbankPay -->> Merchant: Payment Details
            deactivate SwedbankPay
            end
            end

                opt If payment is failed
                activate Payer
                Payer ->> Payer: Event: OnPaymentFailed
                Payer ->>+ Merchant: Check payment status
                deactivate Payer
                Merchant ->>+ SwedbankPay: GET {paymentorder.id}
                deactivate Merchant
                SwedbankPay -->>+ Merchant: rel: failed-paymentorder
                deactivate SwedbankPay
                opt Get PaymentOrder Details (if failed-paymentorder operation exist)
                activate Payer
                deactivate Payer
                Merchant ->>+ SwedbankPay: GET rel: failed-paymentorder
                deactivate Merchant
                SwedbankPay -->> Merchant: Payment Details
                deactivate SwedbankPay
                end
                end
```

{% include alert.html type="informative" icon="info" body="

> Note that in this diagram, the Payer refers to the merchant front-end

Reviewer: ```suggestion
Note that in the above diagram, the Payer refers to the merchant front-end
```

increasing number of payment instrument Swedbank Pay has on offer and
remembering which instrument they prefer and have paid with previously." %}

Swedbank Pay Payment Menu allows your customers to pay with their favorite payment
instruments in just a few simple steps.

## Prerequisites

To start integrating Swedbank Pay Payment Menu, you need the following:

*   [HTTPS][https] enabled web server.
*   Agreement that includes Swedbank Pay Payment Menu.
*   Obtained credentials (merchant Access Token) from Swedbank Pay through
  Swedbank Pay Admin.

## Introduction

To get started with Swedbank Pay Payment Menu, you should learn about its different
components and how they work together. **Payment Menu** authorizes the
payment with our Payment Menu API. The next step is to **Capture** the payment.
You can either capture the total amount, or do a part-capture (as described
under [After Payment][after-payment-capture]). Connect these steps and you have
Swedbank Pay Payment Menu.

{% include iterator.html next_href="payment-order"
                         next_title="Next: Payment Order" %}

[after-payment-capture]: /payment-menu/capture

> [https]: /home/technical-information#connection-and-protocol

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)

> [https]: /home/technical-information#connection-and-protocol

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)

> [https]: /home/technical-information#connection-and-protocol

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)

seamlessly into the merchant website, allowing the payer to choose between the
increasing number of payment instrument Swedbank Pay has on offer and
remembering which instrument they prefer and have paid with previously." %}

Swedbank Pay Payment Menu allows your customers to pay with their favorite payment
instruments in just a few simple steps.

## Prerequisites

To start integrating Swedbank Pay Payment Menu, you need the following:

*   [HTTPS][https] enabled web server.
*   Agreement that includes Swedbank Pay Payment Menu.
*   Obtained credentials (merchant Access Token) from Swedbank Pay through
  Swedbank Pay Admin.

## Introduction

To get started with Swedbank Pay Payment Menu, you should learn about its different
components and how they work together. **Payment Menu** authorizes the
payment with our Payment Menu API. The next step is to **Capture** the payment.
You can either capture the total amount, or do a part-capture (as described
under [After Payment][after-payment-capture]). Connect these steps and you have
Swedbank Pay Payment Menu.

{% include alert.html type="informative" icon="info" body="

> Note that in this diagram, the Payer refers to the merchant front-end

Reviewer: Where is "this diagram"?

(website) while Merchant refers to the merchant back-end." %}

{% include iterator.html next_href="payment-order"
                         next_title="Next: Payment Order" %}

[after-payment-capture]: /payment-menu/capture

> [https]: /home/technical-information#connection-and-protocol

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)

> [https]: /home/technical-information#connection-and-protocol

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)

> [https]: /home/technical-information#connection-and-protocol

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)

> [https]: /home/technical-information#connection-and-protocol

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)

> [https]: /home/technical-information#connection-and-protocol

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)

> [payment-order]: /payment-menu/other-features#payment-orders

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)

> [payment-order]: /payment-menu/other-features#payment-orders

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)

> [payment-order]: /payment-menu/other-features#payment-orders

Reviewer 2: ![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5645371)
